diff --git a/images/production/Containerfile b/images/production/Containerfile
index 684b632..0a6d8cd 100644
--- a/images/production/Containerfile
+++ b/images/production/Containerfile
@@ -1,6 +1,5 @@
-ARG PYTHON_VERSION=3.14.2
-ARG DEBIAN_BASE=bookworm
-FROM python:${PYTHON_VERSION}-slim-${DEBIAN_BASE} AS base
+ARG PYTHON_BASE_IMAGE=python:3.12.11-slim-bookworm@sha256:519591d6871b7bc437060736b9f7456b8731f1499a57e22e6c285135ae657bf7
+FROM ${PYTHON_BASE_IMAGE} AS base
 
 ARG WKHTMLTOPDF_VERSION=0.12.6.1-3
 ARG WKHTMLTOPDF_DISTRO=bookworm
